### ISSUE TITLE
System.IO.IOException on update.xml

### ISSUE THREAD

**Issue by turbanoff** [d2phap/ImageGlass#520]
### System information:
 - Windows OS version: Windows 7 x64.  6.1.7601
 - ImageGlass version: 6.0.12.29

### To Reproduce
1. Open ImageGlass
2. Try to do somethinkg

### Actual behavior:

Exception about unable to access file happened.
```
System.IO.IOException
  HResult=0x80070020
  Сообщение = Процесс не может получить доступ к файлу "C:\Users\user\AppData\Roaming\ImageGlass\Temp\update.xml", так как этот файл используется другим процессом.
  Источник = mscorlib
  Трассировка стека:
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.File.InternalDelete(String path, Boolean checkHost)
   at igcmd.Core.AutoUpdate()
   at igcmd.Program.Main(String[] args)
```

### Expected behavior:
No exceptions.

### Screenshots:
![изображение](https://user-images.githubusercontent.com/741251/57072732-5e7ce780-6ce7-11e9-92f1-a5c5ba914f7e.png)

**Comment by fire-eggs** [comment]
In the short term, un-check off "Check for update automatically" in the Settings window. This should bypass this problem until we have a chance to investigate further.

**Comment by fire-eggs** [comment]
Add exception catching and attempt to prevent file lock (the cause of the error raised).

**Comment by fire-eggs** [comment]
Shipped in V7.0
